cli/args/flags.rs

```diff
@@ -3775,12 +3775,14 @@ fn env_file_arg() -> Arg {
     .help(cstr!(
       "Load environment variables from local file
   <p(245)>Only the first environment variable with a given key is used.
-  Existing process environment variables are not overwritten.</>"
+  Existing process environment variables are not overwritten, so if variables with the same names already exist in the environment, their values will be preserved.
+  Where multiple declarations for the same environment variable exist in your .env file, the first one encountered is applied. This is determined by the order of the files you pass as arguments.</>"
     ))
     .value_hint(ValueHint::FilePath)
     .default_missing_value(".env")
     .require_equals(true)
     .num_args(0..=1)
```

Comment:
should we allow specifying multiple values in a single declaration as well?

Reply:
Hi @crowlKats, before this PR, the top env variable would be applied, for example, if the below was the env file:
```bash
VAR="one"
VAR="two"
```
The `VAR='one'` would take effect in the system, with this PR as well, this will take effect according to order of file meaning, from last to first. Meaning if the above was the last .env file and other preceding file also had `VAR` declared in them then `one` will be applied.

Please suggest if this is the behavior we are looking for or any other.

Reply:
Hi @crowlKats , before this PR the behavior was that _if multiple declaration of the same variable exists in the .env file, the first one was applied._ For example:
```bash
VAR="one"
VAR="two"
```
For the above single file, `VAR='one'` would take effect, and this behavior remains the same but according to order of files passed, that is if the above was the last file passed and other preceding file also had VAR in it, then 'VAR='one'' is applied.

Please suggest, if this is something that we are looking or any anything else.

Reply:
no what i meant multiple files in a single flag call, as in `--env-file=.env.one,.env.two` in addition to allowing multiple calls to `--env-file`

Reply:
hmm, that is achievable using the delimiter flag in clap I guess. Do you want me to update the code @crowlKats ?